### DISCUSSION TITLE
Positions when overlapping.

### DISCUSSION THREAD

**Discussion by A-Winters** [SubtitleEdit/subtitleedit#4911] [Q&A] [answered]
When a sub's timing is overlapped, the positioning all seems to be automatic.
Is there any way I can manually choose whether the next sub line appears above or below the one it has a timing overlap with, please?

**Comment by OmrSi** [comment, marked as answer]
As far as I know, there isn't a way. 
When there is an overlap, the video player usually shows the next line above the previous. I don't think SE can change that behavior. 

What you can do in SE is add a positioning tag to one of the lines like `{\an8}` or `{\an5}`, but the would lift the line way above the other line. 

@niksedk is that accurate?

**Reply by A-Winters** [reply]
Thank you for the response. There's just the occasional part where there's a nice flow of lines fading between line A and line B at the bottom of the screen, but then one will appear above line A every now and then and I end up needing to change the timing to fix it. From what you say, it sounds like a video player thing, rather than an editing thing.

**Reply by OmrSi** [reply]
That might happen if you have 0 gap between the lines. 
You might want to run "Apply minimum gap between line" and setting the gap to 1 ms.

**Reply by A-Winters** [reply]
Thanks.  I have them overlapped intentionally, I just lack control of the position they appear in when I want them all at the bottom of the screen.  By default, the first line will appear at the last line at the bottom.  I'm looking for a way to move text to the second or third from the bottom position so that there's room for the next line to appear under it.

**Reply by OmrSi** [reply]
The only way I see to do this would be by using the tag `{\pos(192,250)}` on the first subtitle to lift it up a bit and leave a place for the second subtitle so it shows up under it.

**Reply by A-Winters** [reply]
This sounds promising. I'll try it and get back to you, thanks 😁
